frontend/index.html

```diff
@@ -2,25 +2,25 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+        <link rel="icon" href="/favicon.ico" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
         <meta
             name="description"
             content="View the scoreboard, schedule meetings and more for the ChessDojo Training Program"
         />
-        <link rel="apple-touch-icon" href="%PUBLIC_URL%/android-chome-192x192.png" />
+        <link rel="apple-touch-icon" href="/android-chome-192x192.png" />
         <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-        <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+        <link rel="manifest" href="/manifest.json" />
         <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
+      Notice the use of  in the tags above.
```

Comment:
This comment doesn't make sense now that `%PUBLIC_URL%` is no longer in the url. Also, why was that removed? Is that only a CRA thing?

Reply:
Will look into this and get back. This file was moved/modified by a conversion tool and I only did a smell test/actual testing.

Reply:
Okay, tested/compared the builds from dev, production, and this PR. This is no longer needed and is populated automatically for us by Vite.

From the migration guide I followed (I refreshed my memory and these steps I did manually):

Vite [automatically resolves](https://vitejs.dev/guide/#index-html-and-project-root) URLs inside index.html, so there's no need for %PUBLIC_URL% placeholders. You can do a search and replace inside your index.html file for this. Be sure to remove all occurrences.